frontend/src/cards/TableCard.tsx

```diff
@@ -75,11 +75,15 @@ export function TableCard(props: TableCardProps) {
       }
     >
       {([queryResponse]) => {
-        const dataWithoutUnknowns = queryResponse.data.filter(
-          (row: Row) =>
-            row[props.breakdownVar] !== UNKNOWN &&
-            row[props.breakdownVar] !== UNKNOWN_RACE
-        );
+        const dataWithoutUnknowns = queryResponse
+          .getValidRowsForFields(
+            metricIds.filter((m) => m !== "brfss_population_pct")
```

Comment:
This doesn't seem like a particularly robust solution to this issue, but I can't think of a relatively simple one right now.. can we open an issue to think about how we can solve this better when we refactor etc?

Reply:
Its the same thing we do for the other cards 🤷‍♂️

Reply:
I don't think we filter for specific metric ids in other cards?

Reply:
Right, I wasn't sure why that wasn't an issue for the other cards.

Reply:
As in why races for which we don't have data show up for other cards? For the disparities per 100k card (I think SimpleBarChartCard.tsx), it doesn't require the population_pct metric so doesn't have this issue. For the disparity bar chart card, it does use the population_pct metric, but it explicitly checks that the primary metric (i.e. % of cases for diabetes) is not null:

![image](https://user-images.githubusercontent.com/3394642/117739855-0faa9800-b1b4-11eb-8b5f-c3a975daaded.png)

One way to make this more robust actually would be to filter out the populationComparisonMetric rather than just the specific "brfss_population_pct" metric, ie change to

getValidRowsForFields(
  metricIds.filter((m) => m !== populationComparisonMetric)

Where populationComparisonMetric is a variable you pull out and save above, in the same function. This would also make this fix work for all data sources rather than just brfss which is generally useful

Reply:
Pushed!